droidlet/lowlevel/hello_robot/remote/utils.py

```diff
@@ -196,7 +196,7 @@ def goto(
         robot.rotate_by(theta_1)
         robot.push_command()
         robot.pull_status()
-        time.sleep(0.1)
+        time.sleep(1)
```

Comment:
any reason for these sleeps? should there be a generic "wait" for a status/command instead?

Reply:
there isn't a generic `wait` available in the base API for stretch.
But I spoke to the Hello folks a few days back and there's a better way to handle this than the sleep.
Will implement in a follow-up PR